### PR TITLE
Update triplea_maps.yaml

### PR DESCRIPTION
**Pull request by sneakingcoward**
game description for waw variants updated

### CODE REVIEW

File: triplea_maps.yaml [hunk 1]
```diff
@@ -938,28 +938,29 @@
   description: |
     <br>This is a MAP SKIN for "Diplomacy"
     <br>A very basic style.  I made this one for people who hate the cursive writing of the other 3 skins.  It has a simple font, no textures, and no cursive.
-  version: 1
+  version: 1.7
```
Comment (ron-murhammer) [inline triplea_maps.yaml]: You updated the wrong version number. This is the version for diplomacy. We are also just incrementing the version number in this file as integer numbers as its the map zip version not the individual maps.

Comment (sneakingcoward) [inline triplea_maps.yaml]: is the update now done ???

Comment (DanVanAtta) [inline triplea_maps.yaml]: Set this number to '2', not '1.7'

Comment (DanVanAtta) [inline triplea_maps.yaml]: This version number will trigger the versions in the map download. That is the only place it is used, to show a map as out of date for download.

Comment (ron-murhammer) [inline triplea_maps.yaml]: @DanVanAtta I pretty sure that is the wrong version number to change.

Comment (DanVanAtta) [inline triplea_maps.yaml]: We have version numbers in the XML file as well. That version number seems to be confused with the one here.  We are due to clean that system up, it is confusing.

The version number in this file is used only on the download screen, nowhere else. The values should be incrementing natural numbers (eg: 1, 2, 3, 4)
The reason for natural numbers here is there is no real meaning behind point releases. The only use for this number is to indicate a new version is available for download. The XML versions are perhaps used further for compatibility notifications.

**Pull request merged**